spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
please explain how this could possibly cause problems.

if we properly coded this to fall back to PyQt4, in case of PyQt5 non-availability, this issue (causing problems on those distributions) doesn’t exist. in this case, we merge this.

if we didn’t, the issue is that we got the fall back wrong and this issue still doesn’t exist. in that case, we fix the fallback code and merge this afterwards.